### DISCUSSION TITLE
Running into install issue.

### DISCUSSION THREAD

**Discussion by james7342** [HotcakesCommerce/hotcakes-commerce-core#449] [Q&A] [answered]
I'm running into an install issue on DNN 9.11.2. I've covered it in the Community Space here. 
https://mmmcommerce.com/Forums/g/Posts/m/1718/Fresh-Install-into-DNN-9-11-2-all-HotCakes-Modules-Generate-Error

Is this a DNN 9.11.2 issue where some DLL's were updated causing the issue? Nothing I've tried from the community pages or my own troubleshooting abilities have been able to identify an RCA.

**Comment by james7342** [comment]
Restored site with DNN 9.10.2 and still same issue.

**Comment by james7342** [comment, marked as answer]
Got it working... Not sure how, why or what, but in comparing DLL's between a working site and non working site. 

DLL version was old.

Non Working:
System.Web.WebPages.dll - Version 3.0.20129.0 Product Version 3.0.1-20129

Working: 
System.Web.WebPages.dll - Version 3.0.61129.10 Product version 3.0.9-61139

**Comment by WillStrohl** [comment]
Despite you somehow finding your way through this issue, we'll take a look at it on our side to see if there is indeed a bug to fix.

**Comment by WillStrohl** [comment]
FYI - The forums at Hotcakes have been migrated to the Community Forums.  Here is the new link:  https://mmmcommerce.com/Forums/aft/350

**Comment by ismaelTartola** [comment]
@james7342 @WillStrohl I have followed all the required steps described in the forum post to reproduce the issue. Everything worked as expected and no reference inconsistency error where detected.

**Reply by james7342** [reply]
@ismaelTartola, Do you have the older DLL version? I think I have a copy of it here for backup. As mentioned not sure how, why, or what placed the older version of the DLL on this particular site(s). I ran into the older DLL version on 2 Managed.com hosted sites. Just replacing the DLL with the version noted above resolved it. The working DLL noted above was from a local DNN instance I have where I do all my local testing before moving to my shared hosting environment. Maybe some with the hosting environment? Not sure.

**Reply by WillStrohl** [reply]
Usually, when there are DLL mismatches like this, someone either installed another module that wasn't using best practices and it was what included it, or someone was performing a "manual" module install/upgrade using FTP and this is what did it.  At least, that's what I've seen the most over the years.

**Reply by james7342** [reply]
@WillStrohl That very well could be the cause. I think these sites originated from back in the DNN 6.X days so they've been around a while. The individual I inherited them from had a tendency install a bunch of what I'll call junk on them. I've since cleaned them up, but this is probably the likely cause as I don't really know what or how they did things. I knew there was something different though as a local desktop install wasn't having the issue. Just took some time to track down what the cause was.

Thanks for looking into though. And hopefully if someone runs into this problem this discussion might help them get it resolved.
